docs/content/dagster-cloud/account/managing-user-agent-tokens.mdx

```diff
@@ -27,9 +27,19 @@ Agent tokens are used to authenticate [Hybrid agents](/dagster-cloud/deployment/
 
 After the token is created:
 
-- **To edit a token's description**, click the **pencil icon**.
 - **To view a token**, click **Reveal token**. Clicking on the token value will copy it to the clipboard.
-- **To revoke a token**, click **Revoke**.
+- **To edit a token's description**, click the **pencil icon**.
+
+### Modifying agent tokens
```

Comment:
I think it'd be helpful to re-name this section to something about granting permissions - maybe "Assigning agent token permissions" or something along those lines. That's more specific than 'modifying,' and users can only assign permissions or revoke the token in this dialog at the moment anyhow.